### PR TITLE
Convert all packages to typescript

### PR DESCRIPTION
**Pull request by mharsat**

### CODE REVIEW

File: packages/plugin-image/web/src/typeMapper.js [hunk 1]
```diff
@@ -11,6 +10,6 @@ const imageRenderDescriptor = {
 };
 
 export const typeMapper = () => ({
-  [IMAGE_TYPE_LEGACY]: imageRenderDescriptor,
-  [IMAGE_TYPE]: imageRenderDescriptor,
+  IMAGE: imageRenderDescriptor,
+  'wix-draft-plugin-image': imageRenderDescriptor,
```
Comment (Barackos) [inline packages/plugin-image/web/src/typeMapper.js:14]: Why the change?

Comment (Barackos) [inline packages/plugin-image/web/src/typeMapper.js:14]: Why the change?

File: packages/plugin-video/web/package.json [hunk 2]
```diff
@@ -18,6 +18,8 @@
     "dist"
   ],
   "main": "dist/module.cjs.js",
+  "types": "dist/types/src/index.d.ts",
+  "types": "dist/types/src/index.d.ts",
```
Comment (Barackos) [inline packages/plugin-video/web/package.json:22]:
```suggestion
  "types": "dist/types/src/index.d.ts",
```

File: packages/plugin-image/web/src/typeMapper.js (hunk 1 above)
Comment (mharsat) [reply packages/plugin-image/web/src/typeMapper.js:14]: Solved a weird issue "cannot use external module name as key", I followed the pattern done in other `typeMapper.js` files that were converted to flow like [video](https://github.com/wix-incubator/rich-content/blob/master/packages/plugin-video/web/src/typeMapper.js)

File: packages/plugin-video/web/package.json (hunk 2 above)
Comment (mharsat) [reply packages/plugin-video/web/package.json:22]: nice!
